region/maridia/outer.json

```diff
@@ -166,79 +166,91 @@
           "from": 2,
           "to": [
             {
-              "id": 1,
-              "requires": [ "MaridiaTubeBroken" ],
-              "unlock": null
-            },
-            {
-              "id": 3,
-              "requires": [ "MaridiaTubeBroken" ],
+              "id": 7,
+              "requires": null,
               "unlock": null
-            },
+            }
+          ]
+        },
+        {
+          "from": 3,
+          "to": [
             {
-              "id": 4,
-              "requires": [
-                "MaridiaTubeBroken",
-                {"or": [
-                  "canSwim",
-                  "HiJump",
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "canSpringBallJump"
-                  ]}
-                ]}
-              ],
+              "id": 5,
+              "requires": null,
               "unlock": null
-            },
+            }
+          ]
+        },
+        {
+          "from": 4,
+          "to": [
             {
-              "id": 5,
-              "requires": [ "canUsePowerBombs" ],
-              "yields": [ "MaridiaTubeBroken" ],
+              "id": 8,
+              "requires": null,
               "unlock": null
             }
           ]
         },
         {
-          "from": 3,
+          "from": 5,
           "to": [
             {
               "id": 1,
               "requires": null,
               "unlock": null
             },
             {
-              "id": 2,
-              "requires": [ "MaridiaTubeBroken" ],
+              "id": 3,
+              "requires": null,
               "unlock": null
             },
             {
               "id": 4,
+              "requires": [
+                "MaridiaTubeBroken",
+                "canSuitlessMaridia",
+                {"or":[
+                  {"adjacentRunway": {
+                    "fromNode": 1,
+                    "usedTiles": 1
+                  }},
+                  {"adjacentRunway": {
+                    "fromNode": 3,
+                    "usedTiles": 1
+                  }}
+                ]}
+              ],
+              "unlock": null,
+              "note": "This direct link involves using an adjacent room without water physics to initiate a jump that takes Samus through the top door. The runway is to get enough speed to clear the edge of the tube."
```

Comment:
Devs need to know that they can't use adjacentRunways with "canSwim" reqs for this one somehow.

Reply:
That's a damn good point. I have no idea how this could be done cleanly. Should I just take this out of logic until we figure it out?

Reply:
One thing we could do is straight-up put a `submerged` property on our doors, then I imagine we could also put the `submerged` property into the `adjacentRunway` object, where a default value of `null` would mean "doesn't matter".

Is that too specific?

Reply:
I think it just have to go. Too much work for seemingly one occurrence.

Reply:
I think your original point remains valid. But you're also right, luckily in this case having Gravity does make this strat redundant. I'll add something to the note to explain this.